ui/app/templates/components/masked-input.hbs

```diff
@@ -1,12 +1,12 @@
 <div class="masked-input {{if shouldObscure "masked"}} {{if displayOnly "display-only"}} {{if allowCopy "allow-copy"}}" data-test-masked-input>
 	{{#if displayOnly}}
 		<pre class="masked-value display-only is-word-break">{{displayValue}}</pre>
-	{{else}}
+  {{else}}
   <textarea
     class="input masked-value"
     rows=1
     wrap="off"
-    placeholder="value"
+    placeholder={{value}}
```

Comment:
Do we want to display the value passed in as the placeholder - does that reveal the value? Maybe we should just remove the placeholder value entirely. I think this maybe was left in from some copying where masked-input was used the first time - the kv edit form in secrets so there was a "key" and a "value".